librus/__init__.py

```diff
@@ -180,21 +180,22 @@ def list_messages(self, get_content=False):
 
         for row in response.html.find('.stretch > tbody > tr'):
             cells = row.find('td')
+            href = cells[3].find('a')[0].attrs['href']
             message = Message(
+                message_id=href.strip()[-10:-3],  # https://synergia.librus.pl/wiadomosci/1/1/0000000/f0, id = 0000000
```

Comment:
Thanks for the example. I also noticed you changed `-9:-3` to `-10:-3`
This suggests the offset-based approach may or may nor work. e.g. Librus starts to generate IDs that are 1 digit longer.

i propose to 
- extract all numbers that are 5 digits or more (`r'\d{5,}'` or something like that)
- verify this is just one such match and fail otherwise
- treat it as the id (keeping it in string form)

Reply:
Or maybe just split on `/` and take second to last element?